nncf/torch/quantization/layers.py

```diff
@@ -214,7 +214,6 @@ def __init__(self, unified_scale_groups, shared_input_operation_set_groups):
         super().__init__()
         self.unified_scale_groups = unified_scale_groups
         self.shared_input_operation_set_groups = shared_input_operation_set_groups
-        self.quantization_points = {}  # type: Dict[QuantizationPointId, PTQuantizationPoint]
```

Comment:
Why it was removed?

Reply:
This attribute is being set inside `super().__init__()` and here it's reinitialized again